### DISCUSSION TITLE
how to set Custom header with getPresignedObjectUrl() minio java

### DISCUSSION THREAD

**Discussion by Kim-Robin** [minio/minio#18195] [Q&A]
I am using java to request getPresignedObjectUrl() with header something like ‘Range: bytes=1-300’. I have used .extraHeaders(header) but I am getting “There were headers present in the request which were not signed” with AccessDenied.

I am just trying to get partial contents of the file. How should I fix this issue?

Should I configure on minio server somehow or should I add something extra to my code before getPresignedObjectUrl() call?

**Comment by lenzDocthread** [comment]
any updates on this? would like to find a way too

**Comment by klauspost** [comment]
Add issue on https://github.com/minio/minio-java/issues with reproducer code.
